<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDialog</class>
 <widget class="QDialog" name="ChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="rotationLabel">
       <property name="text">
        <string>Угол поворота</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rotationEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="perimeterLabel">
       <property name="text">
        <string>Периметр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="perimeterOutput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Длина             </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Ширина          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" rowspan="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="areaLabel">
       <property name="text">
        <string>Площадь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="areaOutput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="submitDialogBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>submitDialogBox</sender>
   <signal>accepted()</signal>
   <receiver>ChangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitDialogBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
